onnxscript/evaluator.py

```diff
@@ -144,3 +204,99 @@ def call_ort(schema, args, kwargs, implicit_args=None):
     # Map ORT output values to the onnxscript representation-type.
     cast_result = [ort_to_os_value(x) for x in result]
     return cast_result[0] if len(cast_result) == 1 else cast_result
+
+
+def id(schema):
```

Comment:
You should use another name. `id` is a built-in function.

Reply:
done